src/Reflector/Generic.php

```diff
@@ -41,33 +41,18 @@ public function __construct(SourceLocator $sourceLocator)
      * specified and returns the \Reflector.
      *
      * @param Identifier $identifier
-     * @param bool $autoReflectInternals
+     *
      * @return Reflection
      */
-    public function reflect(Identifier $identifier, $autoReflectInternals = true)
+    public function reflect(Identifier $identifier)
     {
         $aggregate = $this->sourceLocator;
```

Comment:
If the "aggregate" logic has been abstracted into the `AggregateSourceLocator` (which is better) this should be named more appropriately (it is not definitely an `Aggregate` any more)